rfcs/react-components/convergence/semantic-shared-colors.md

```diff
@@ -0,0 +1,161 @@
+- [7 **status** shared colors](https://github.com/microsoft/fluentui/blob/76e6598eae0c9247fb7f8834c1e24d8e864bf5df/packages/tokens/src/types.ts#L379-L388), where each contains [9 color slots](https://github.com/microsoft/fluentui/blob/76e6598eae0c9247fb7f8834c1e24d8e864bf5df/packages/tokens/src/types.ts#L159-L169) (plus 3 of them contain 10th slot).
+
+There is an overlap between the two groups - `red` and `marigold` colors from shared color tokens are used as persona shared colors as well. That is intentional, for that reasons, the "persona slots" have the same color mapping in both groups:
+
+| Color slot                          | Notes                 |
+| ----------------------------------- | --------------------- |
+| `colorPaletteRedBackground1`        | status                |
+| `colorPaletteRedBackground2`        | status + persona      |
+| `colorPaletteRedBackground3`        | status                |
+| `colorPaletteRedForeground1`        | status                |
+| `colorPaletteRedForeground2`        | status + persona      |
+| `colorPaletteRedForeground3`        | status                |
+| `colorPaletteRedForegroundInverted` | Red, Green and Yellow |
+| `colorPaletteRedBorderActive`       | status + persona      |
+| `colorPaletteRedBorder1`            | status                |
+| `colorPaletteRedBorder2`            | status                |
+
+This results in 150 (28 _ 3 + 7 _ 9 + 3) color tokens.
+
+Adding new tokens comes at a [runtime performance cost](https://github.com/microsoft/fluentui/blob/master/rfcs/react-components/convergence/theme-shape.md).
+
+## Problem statement
+
+There is no semantic meaning in the current shared color tokens. The intended use case is communicated only by a design spec. See [#26168](https://github.com/microsoft/fluentui/issues/26168), [#25720](https://github.com/microsoft/fluentui/issues/25720).
+Designers plan to update the color values for the status shared colors - that might seem just like a hex value change, but that results in a color name change: for example **red** will be replaced by **cranberry**.
+
+### Token change request from design
+
+| Current name | Meaning      | New name  | Color change                                 |
+| ------------ | ------------ | --------- | -------------------------------------------- |
+| `red`        | danger, busy | ???       | Change `red` to `cranberry`                  |
+| `green`      | success      | success   | No change                                    |
+| `darkOrange` | severe       | REMOVE    | Remove `darkOrange` and "severe" as a status |
+| `yellow`     | warning      | warning   | Change `yellow` for `orange`                 |
+| `berry`      | OOF          | oof       | No change                                    |
+| `lightGreen` | available    | available | No change                                    |
+| `marigold`   | away         | away      | No change                                    |
+
+_‼️ TODO: verify with design_
```

Comment:
NOTE: after discussing the topic with @daisygeng, the current Design requirement is to introduce semantic color names only for **danger**, **success** and **warning**. I will be updating the RFC accordingly.

Reply:
Updated the spec accordingly - if we ignore presence tokens, introducing just `status` tokens now shows different numbers (of tokens eventually added/affected).